projects/uci-console/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@samagra-x/uci-console",
-  "version": "1.0.0-beta.3",
+  "version": "1.1.3",
```

Comment:
This can be release 1.0.0 right?

Reply:
No, 1.1.1 is already live on NPM so we cannot downgrade it. The current version is 1.1.3 (I pushed it yesterday).